truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -148,6 +144,11 @@ def RbConfig.expand(val, config = CONFIG)
     val.replace(newval) unless newval == val
     val
   end
+
+  CONFIG.each_value do |val|
+    RbConfig::expand(val)
+  end
```

Comment:
> @eregon I think there are less chances for typos to use the MAKEFILE_CONFIG interpolation format to set the CONFIG values over string interpolation.

What do you mean?
`"#{prefix}/lib/#{ruby_version}"` is the same as
`"$(prefix)/lib/$(ruby_version)"` to me, only less efficient.